bin/log.js

```diff
@@ -0,0 +1,103 @@
+const colors = {
+	Reset: "\x1b[0m",
+	Bright: "\x1b[1m",
+	Dim: "\x1b[2m",
+	Underscore: "\x1b[4m",
+	Blink: "\x1b[5m",
+	Reverse: "\x1b[7m",
+	Hidden: "\x1b[8m",
+	FgBlack: "\x1b[30m",
+	FgRed: "\x1b[31m",
+	FgGreen: "\x1b[32m",
+	FgYellow: "\x1b[33m",
+	FgBlue: "\x1b[34m",
+	FgMagenta: "\x1b[35m",
+	FgCyan: "\x1b[36m",
+	FgWhite: "\x1b[37m",
+	BgBlack: "\x1b[40m",
+	BgRed: "\x1b[41m",
+	BgGreen: "\x1b[42m",
+	BgYellow: "\x1b[43m",
+	BgBlue: "\x1b[44m",
+	BgMagenta: "\x1b[45m",
+	BgCyan: "\x1b[46m",
+	BgWhite: "\x1b[47m",
+};
+
+class logger {
+	constructor(title) {
+		this.title;
+		process.stdout.write(`${colors.Bright}${title} - ${colors.FgCyan}webpack-cli ${colors.Reset}\n`);
+		return;
+	}
+
+	log(message) {
+		const messages = message.split("\n");
+		if (messages.length === 1) {
+			process.stdout.write(`${colors.Bright} • ${colors.Reset}${messages[0]} \n`);
```

Comment:
I don't know but current syntax is also modular and readable. Plus logging requirements of webpack-cli (not saying about webpack build outputs ) are very minimal ( even the logging of the questions is governed by inquirer not chalk ). We can definitely avoid chalk without disturbing our eco-system in this case.

P.S.
I am not against usage of third party packages 😅.

Reply:
Well I found out inquirer has chalk as direct dependency. So, this might not serve the good.
So if current codebase seems to be a mess to all.
I will rewrite it to chalk 🤓

Reply:
Used chalk now thanks for this review 😎